File: module1/#2Bien_KieuDuLieu_ToanTu/ThucHanh/ToanTu.js
let inputWidth;
let inputHeight;

inputWidth = prompt('Enter the Width :');
inputHeight = prompt('Enter the Height');

let Width = parseInt(inputWidth);
let Height = parseInt(inputHeight);

let area = Width * Height;//tinh dien tich hinh chu nhat
document.writeln('The area is :' + area);

File: module2/#1.NgonNguLapTrinhJava/src/ThucHanh/ToanTu.java
package ThucHanh;

import java.util.Scanner;

public class ToanTu {

    public static void main(String[] args) {
        float width;
        float height;

        Scanner scanner = new Scanner(System.in);

        System.out.printf("Enter width: ");
        width = scanner.nextFloat();

        System.out.printf("Enter height: ");
        height = scanner.nextFloat();

        float area = width * height;

        System.out.println("Area is: " + area);
    }
}
